seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/service/impl/UserServiceImpl.java

```diff
@@ -142,6 +145,10 @@ public UserSimpleInfoRes login(UserLoginReq req) {
         if (Objects.isNull(user)) {
             throw new SeatunnelException(USERNAME_PASSWORD_NO_MATCHED);
         }
+
```

Comment:
Use interceptors because you need to verify that the user state is disabled on any interface

Reply:
done.